src/test/ui/inference/issue-71309.stderr

```diff
@@ -0,0 +1,15 @@
+error[E0308]: `?` operator has incompatible types
+  --> $DIR/issue-71309.rs:2:18
+   |
+LL |     let y: u32 = x?;
+   |                  ^^ expected `u32`, found `i32`
+   |
+   = note: `?` operator cannot convert from `i32` to `u32`
+help: you can convert an `i32` to a `u32` and panic if the converted value doesn't fit
+   |
+LL |     let y: u32 = x.try_into().unwrap();
+   |                  ^^^^^^^^^^^^^^^^^^^^^
```

Comment:
For some reason [this suggestion is fine in stable](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=0bac423871ead09f59a80107beeb618d) and nightly, but it isn't here.